WooCommerce/UITestsFoundation/Screens/Login/PrologueScreen.swift

```diff
@@ -1,31 +1,23 @@
 import ScreenObject
 import XCTest
 
-private struct ElementStringIDs {
-    static let continueButton = "Prologue Continue Button"
-    static let siteAddressButton = "Prologue Self Hosted Button"
-}
```

Comment:
> Hmm, this is strange, since the Screenshot test uses an instance of tabBar in OrdersScreen. It should fail, because once tabBar is removed, Xcode will not find such a class member. Are you able to build the WooCommerceScreenshots target with tabBar removed? (for me, it failed even in WooCommerceUITests target, but I'd expect it to fail in WooCommerceScreenshots for sure)

Ah right, I did not test `WooCommerceScreenshots` as target as that is known to be failing and have not looked into fixing that yet. `WooCommerceUITests` passed without issue for me in multiple runs. I've added it back since it is causing issues on some local envs.

Reply:
> Even in this short screen object file, we can already see the same string twice (`Prologue Continue Button`), and this might become more evident in complex screen objects with a higher number of methods.

Yeah personally, I am used to this style having group all on top of the file and potentially having it be reusable (more apparent on "busier" screens). However, we are moving away from this style in WPiOS and thought having some some consistency between the two apps would be great. 

We have not however, discussed it properly, which we should. In this case, my bad for jumping the gun on making the change. I'll get the conversation started.